Automattic/jetpack
# Build: Fix the `yarn build-production` script

We're not using `yarn build-production` for the build process in any way, but it's confusing to see the command there and see it fail.

Turned out that some dependency of dops-components does not install its dependencies well if `yarn` is run when `NODE_ENV=production` is set. So, this PR makes `yarn` run normally and sets the `NODE_ENV=production` value after that, just when building the client.

#### Changes proposed in this Pull Request:

* Updates `package.json` to include a script `build-production-client`.
* Updates `package.json` to make the `build-production` script rely on `build-production-client`.

#### Testing instructions:

* Clone this branch.
* Run `yarn clean-client && yarn cache clean && yarn build-production`
* Expect everything to run well.
